src/types/durableClientTypes.ts

```diff
@@ -1,9 +1,229 @@
-import { FunctionInput } from "@azure/functions";
+import { FunctionInput, HttpRequest, HttpResponse } from "@azure/functions";
+import { EntityId, EntityStateResponse, HttpManagementPayload, IHttpResponse } from "../classes";
+import { DurableOrchestrationStatus } from "../durableorchestrationstatus";
+import { IHttpRequest } from "../ihttprequest";
+import { OrchestrationRuntimeStatus } from "../orchestrationruntimestatus";
+import { PurgeHistoryResult } from "../purgehistoryresult";
 
 export interface DurableClientInput extends FunctionInput {
     type: "durableClient";
 }
 
+/**
+ * Client for starting, querying, terminating and raising events to
+ * orchestration instances.
+ */
+export declare class DurableClient {
+    /**
+     * The name of the task hub configured on this orchestration client
+     * instance.
+     */
+    readonly taskHubName: string;
+
+    /** @hidden */
+    public readonly uniqueWebhookOrigins: string[];
```

Comment:
Any particular reason you added this back? I guess I don't fully understand how the "hidden" tag works, but I assumed as a part of manually generating the types that we wouldn't see any hidden fields

Reply:
Yes, the reason was I found this referenced in the tests, so I figured there might be some use-case where this is needed for some reason. I'm not sure how the `@hidden` tags work either, but to not change the old behavior, I kept the `@hidden` tags.

Reply:
Also, per your other suggestions, shouldn't we lean on just keeping them as-is for now to match the auto-generated declarations, and remove them in a separate change if we decide to?

Reply:
I was lumping hidden fields together with private fields in my head. Both are technically accessible from JavaScript regardless, but the auto-generated types made it pretty clear they shouldn't be used. If you feel we should leave hidden tags in, I'm okay with that

Reply:
I think the difference is `@hidden` fields can be accessed by TypeScript code too. It's just hidden from documentation. I'm okay removing `@hidden` fields in general, but I'd also mark that as a breaking change. This particular field though I think we should keep because I found it used in the `getClient()` tests here: https://github.com/Azure/azure-functions-durable-js/blob/4b4eae8a5dc9b1571be6cdfda229b7ae2d0fba4a/test/unit/getclient-spec.ts#L101 

I'm not sure if this was supposed to be testing some internal thing, or if this in itself is a valid use-case, but either way, it feels breaking to remove